src/focus-trap-react.js

```diff
@@ -18,6 +26,19 @@ class FocusTrap extends React.Component {
     }
   }
 
+  getChildContext() {
+    return {
+      _FocusTrapReact: {
+        pause: () => this.focusTrap.pause(),
+        unpause: () => {
+          if (this.props.paused === false) {
```

Comment:
I think we could leave out this condition, since focus-trap already handles it: https://github.com/davidtheclark/focus-trap/blob/master/index.js#L93